app/src/pages/embedding/ExportSelectionButton.tsx

```diff
@@ -25,6 +30,40 @@ type ExportInfo = {
   fileName: string;
 };
 
+const EXPORTS_CODE_SNIPPET = `import phoenix as px
+
+px.active_session().exports`;
```

Comment:
maybe add a line to show how to access the -1 and to view the dataframe

Reply:
addressed in a1dd8447